homeassistant/components/knx/services.yaml

```diff
@@ -2,47 +2,87 @@ send:
   description: "Send arbitrary data directly to the KNX bus."
   fields:
     address:
+      name: "Group address"
       description: "Group address(es) to write to."
+      required: true
       example: "1/1/0"
+      selector:
+        text:
     payload:
+      name: "Payload"
```

Comment:
As this now uses selectors, only fields with a selector are shown. Payload does not have a selector now, thus hidden. What that on purpose? Why not add an `object` selector instead?

Reply:
It expects int, string or a list of ints (<256) so I guess object would be the way to go. 
Advanced would not make sense as it is a required field.

Reply:
Right, well in that case, if required, this schema is now problematic, so that needs fixing.